Gopkg.toml

```diff
@@ -76,8 +76,10 @@ required = [
   source = "github.com/openshift/kubernetes-client-go"
 
 [[override]]
+  # TODO: roll back to use https://github.com/kubernetes-sigs/controller-runtime/tree/release-0.1 once https://github.com/kubernetes-sigs/controller-runtime/pull/410 merged
   name = "sigs.k8s.io/controller-runtime"
-  version = "v0.1.10"
+  source = "github.com/ingvagabund/controller-runtime"
```

Comment:
https://github.com/kubernetes-sigs/controller-runtime/pull/410 got merged, please switch back to the upstream source of `release-0.1` branch.

Reply:
oh great, will do it now

Reply:
done